core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -100,12 +100,6 @@ oppia.factory('StatsReportingService', [
         });
     };
 
-    if (!_editorPreviewMode) {
-      $interval(function() {
-        postStatsToBackend();
-      }, 10000);
-    }
-
     var postStatsToBackend = function() {
```

Comment:
I think it's worth adding a comment here explaining when this is called.

Reply:
Done.